cypress/integration/08.SocialMediaAccounts.feature

```diff
@@ -0,0 +1,14 @@
+Feature: List Social Media Accounts
+  As a User
+  I'd like to enter my social media accounts
+  So I can show them to other users to get in contact
+
+  Background:
+    Given I have a user account
+    And I am logged in
+    And I am on the "settings" page
+
+  Scenario: Adding Social Media Acccounts
+    Given I click on the "My social media accounts" link
+    Then I should be on the "/settings/my-social-media-accounts" page
+    And I should be able to add a social media link
```

Comment:
@Kachulio1 could you add another assertion to see the social media account link on the page? The [video](https://dashboard.cypress.io/#/projects/qa7fe2/runs/1147/specs) stops before we can see how the link appears.

Reply:
at the moment, the link doesn't appear, we are still working on that implementation